obspy/io/css/ChangeLog.txt

```diff
@@ -0,0 +1,14 @@
+
+package obspy.io.css
+====================
+
+2020-10-28 pz
+Open CSS binary waveform files compressed with gzip.
```

Comment:
In terms of the changelog, I think @megies meant rather a one line in the `CHANGELOG.txt` located in main directory of repo :)

Reply:
Thanks for advice. Done.